.github/workflows/downstream_tests.yml

```diff
@@ -87,21 +87,16 @@ jobs:
       - name: show-deps
         run: uv pip freeze
       - name: Create assets directory, copy over index.html
+        continue-on-error: true
```

Comment:
`continue-on-error` just because i dont think this is needed in tests, but i doesnt hurt to have in case we do need it later